quarkus/service/src/test/java/org/apache/polaris/service/quarkus/catalog/IcebergCatalogTest.java

```diff
@@ -1846,4 +1847,28 @@ public void testConcurrencyConflictUpdateTableDuringFinalTransaction() {
   private static InMemoryFileIO getInMemoryIo(IcebergCatalog catalog) {
     return (InMemoryFileIO) ((ExceptionMappingFileIO) catalog.getIo()).getInnerIo();
   }
+
+  @Test
+  @Disabled
+  @Override
+  public void createAndDropEmptyNamespace() {
+    // TODO: remove this when quarkus bug : https://github.com/quarkusio/quarkus/issues/46642
```

Comment:
This is a won't-fix.
You have to tell it to use the Junit5-exception.
Cf [here](https://github.com/projectnessie/nessie/blob/main/versioned/tests/src/main/java/org/projectnessie/versioned/tests/Junit5Assumptions.java) and [here](https://github.com/projectnessie/nessie/blob/main/versioned/tests/src/main/resources/META-INF/services/org.assertj.core.configuration.Configuration)

Reply:
This is nice, thanks @snazy let me try this out.

Reply:
@snazy  i tried this unfortunately this is not working or i might be missing something 
![Screenshot 2025-04-24 at 10 55 24 AM](https://github.com/user-attachments/assets/4080fb77-742a-4df5-a63f-2725374baa3e)

Though if we have 

```
  static {
    Assumptions.setPreferredAssumptionException(PreferredAssumptionException.JUNIT5);
  }
```

my read is they are doing same thing, let me debug this more, removed this for meanwhile